node/cmd/guardiand/node.go

```diff
@@ -1331,34 +1349,37 @@ func runNode(cmd *cobra.Command, args []string) {
 		var solanaFinalizedWatcher *solana.SolanaWatcher
 		if shouldStart(solanaRPC) {
 			logger.Info("Starting Solana watcher")
-			readiness.RegisterComponent(common.ReadinessSolanaSyncing)
+			readinessSync := common.ChainIdToReadinessSyncing(vaa.ChainIDSolana)
+			readiness.RegisterComponent(readinessSync)
```

Comment:
It's safe to do `RegisterComponent` in ` ChainIdToReadinessSyncing` and remove the necessary registration everywhere. Register is currently *not* safe to be called multiple times but since we own the package that can be changed.

Reply:
Added a separate method to generate the readiness component and register it.